### ISSUE TITLE
[Bug]: FCM Connected but no updates in HA

### ISSUE THREAD

**Issue by rikman122** [bvis/aegis-hass#234]
### Describe the bug

I've setup the latest (1.8.0) integration following the README with FCM configured and I can see it's connected under System Information:

<img width="358" height="230" alt="Image" src="https://github.com/user-attachments/assets/00660730-2fbb-424f-95f7-08348b708eac" />

I can arm/disarm from HA and it's instantly updated on the AJAX app but the other way around is not working. Eventually I get the updates from AJAX app to HA from the poll mechanism but not from FCM push.

What I also noticed is that after an HTS connection error the same set of logs lines appears after arming/disarming in the app:

`2026-06-02 14:41:41.548 DEBUG (MainThread) [custom_components.aegis_ajax.coordinator] Manually updated aegis_ajax data
2026-06-02 14:41:41.549 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Skipping unsupported device type: None
2026-06-02 14:41:41.549 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Unsupported LightDevice (7b) wire-shape: f5=bytes(5)
2026-06-02 14:41:41.549 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Unsupported LightDevice (7b) bytes: 2a051203355f30
2026-06-02 14:41:46.465 DEBUG (MainThread) [custom_components.aegis_ajax.coordinator] Manually updated aegis_ajax data
2026-06-02 14:41:46.466 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Skipping unsupported device type: None
2026-06-02 14:41:46.466 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Unsupported LightDevice (7b) wire-shape: f5=bytes(5)
2026-06-02 14:41:46.466 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Unsupported LightDevice (7b) bytes: 2a051203355f30
2026-06-02 14:41:49.993 DEBUG (MainThread) [custom_components.aegis_ajax.coordinator] Manually updated aegis_ajax data
2026-06-02 14:41:49.994 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Skipping unsupported device type: None
2026-06-02 14:41:49.994 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Unsupported LightDevice (7b) wire-shape: f5=bytes(5)
2026-06-02 14:41:49.994 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Unsupported LightDevice (7b) bytes: 2a051203355f30
2026-06-02 14:41:53.492 DEBUG (MainThread) [custom_components.aegis_ajax.coordinator] Manually updated aegis_ajax data
2026-06-02 14:41:53.493 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Skipping unsupported device type: None
2026-06-02 14:41:53.494 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Unsupported LightDevice (7b) wire-shape: f5=bytes(5)
2026-06-02 14:41:53.494 DEBUG (MainThread) [custom_components.aegis_ajax.api.devices] Unsupported LightDevice (7b) bytes: 2a051203355f30`

Full Logs: 

[home-assistant_2026-06-02T12-31-22.096Z.log](https://github.com/user-attachments/files/28508432/home-assistant_2026-06-02T12-31-22.096Z.log)

### Steps to reproduce

1. Add the integration
2.  Configure FCM
3. Arm/Disarm from HA immediately reflects in the AJAX app
4. Arm/Disarm from AJAX app doesn't reflect in HA

### Expected behavior

Arm/Disarm from AJAX app immediately reflects in HA when FCM is correctly configured

### Home Assistant version

026.5.4

### Integration version

Other / not listed

### Installation method

HACS

### App label (co-branded app)

Ajax

### FCM push notifications configured?

Yes

### Relevant logs

```text

```

**Comment by bvis** [comment]
Thanks for the detailed log — it actually rules a few things out. FCM connects fine (`Successfully logged in to MCS endpoint`), and the device stream + hub link are both working. But across the whole ~11-minute capture there's **not a single inbound push** logged (every received push prints a `Push notification received` line, and none appear), which means the log doesn't contain an actual arm/disarm-from-app event to diagnose — it shows the integration healthy and waiting.

One clarification: the `Unsupported LightDevice (7b)` lines you flagged are **not** the arm/disarm state. That's a separate device of yours (looks like a socket/outlet family) that the integration doesn't fully decode yet — harmless noise that happened to line up in time. Arm/disarm state doesn't flow through those.

To pin down where the push path breaks, could you grab a **targeted** capture:
1. Keep debug logging on (`custom_components.aegis_ajax: debug`).
2. Note the wall-clock time, then **arm or disarm from the Ajax app** and wait ~30s.
3. Paste the log lines from that moment.

What I'm looking for: whether a `Push notification received` line fires within a few seconds of your action. If it does, the push is arriving and we're dropping it (and I can see why). If it never appears, the push isn't being delivered to this client at all — a different problem. Either way that single capture tells me which path to chase. I'd rather not guess at a fix until I can see the actual event.

**Comment by bvis** [comment]
One likely cause worth ruling out first, before the capture: **are you using the same Ajax account in the phone app and in the Home Assistant integration?**

That would fit your symptoms exactly. Commands *from* HA work because they're a direct request (nothing to do with push), but Ajax's event push doesn't reliably fan out to two sessions of the same account — the actively-used phone app tends to "own" the push channel, so the integration registers its token (which your log shows it does) yet never receives the events. Polling still picks the change up later, which is what you're seeing.

If that's your setup, the fix is a **dedicated Ajax account for HA**:
1. In the Ajax app → your space → **Users → Invite**, create/invite a separate account (a non-admin **User** role is enough to read state and arm/disarm).
2. Reconfigure the integration to log in with that account.
3. Then arm/disarm from the phone app and check whether HA updates instantly.

This is the recommended setup anyway (see the "dedicated, limited Ajax account" note in the README). If you're **already** on separate accounts, tell me — then the targeted capture from my previous comment is the way to go, and I'll chase it from the push-delivery side.

**Comment by rikman122** [comment]
Hi! Thanks for the quick response. In fact, I was using a different account but it turned out that at least in my case the account needs admin role to work.

Thank you again for your amazing work! you can close this ticket

**Comment by andrea271988** [comment]
I've always had the same account and it worked, but with recent updates I've noticed the same problem. Creating a new AJAX account requires a new phone number, which I don't have.

**Comment by rikman122** [comment]
yeah it's a pity that you need a phone number for new accounts. What I did was to use the phone number attached to the hub's SIM card. If you are also using a SIM on yours maybe you can do the same.

**Comment by bvis** [comment]
Thanks both for the detective work — this turned out to be account-side rather than something the integration can change:

- Home Assistant works best with its **own** Ajax account (not the one on your phone), and that account must be **allowed to receive event notifications** — a very limited role can connect to FCM yet never get events (@rikman122 found a full-access role did the trick).
- New Ajax accounts needing a phone number is an Ajax limitation; @rikman122's tip of using the hub's SIM number is a good workaround.

I've updated the README to spell both of these out so others don't hit the same wall. Closing since there's nothing actionable on the integration side here — but if any of you spot something that does look like a bug or a possible improvement (e.g. the "same account used to work and recently stopped" case), please don't hesitate to open a new issue with logs and we'll dig in. Thanks again! 🙏

**Comment by andreaugg94** [comment]
Hi @bvis, I've been using this integration with the same Ajax app account since the beginning, and it worked before.
What changed that made it necessary to use a different account to ensure status updates?
